pkgs/applications/graphics/json-plot/default.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoPackage,
+}: let
+  repo = "jp";
+  version = "1.1.12";
+  owner = "sgreben";
+in
+  buildGoPackage {
```

Comment:
Please use buildGoModuel

Reply:
upstream does not support modules, I have opened up an issue (sgreben/jp#29), but my understanding was that we should aim to mimic upstream dependency resolution as closely as possible to prevent "breaks on nix" issues?

Reply:
> my understanding was that we should aim to mimic upstream dependency resolution as closely as possible to prevent "breaks on nix" issues?

Yes and no. For go we do that but for example for python and most other things we ignore those if the package still works for practical reasons. But since upstream is pretty much dead we can use goPackagePath here.